src/database/migrations/20190708134018-create-article.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Articles', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      description: {
+        type: Sequelize.STRING
```

Comment:
The reason for allowing the `description` to a datatype of `STRING` during implementation was due to the fact that it is expected that the Application should have a description whose content should not be allowed to be as long as that of the article body itself.
However, I know that your suggestion is valid, this can also be achieved through the frontend where it will make sense to have a validation to control the number of characters a user will input as `description`